Sources/BraintreePayPal/BTPayPalClient.swift

```diff
@@ -183,6 +183,46 @@ import BraintreeDataCollector
         }
     }
 
+    /// Tokenize a PayPal request to be used with the Edit FI flow.
+    ///
+    /// On success, you will receive an instance of `BTPayPalAccountNonce`; on failure or user cancelation you will receive an error.
+    /// If the user cancels out of the flow, the error code will be `.canceled`.
+    ///
+    /// - Parameters:
+    ///   - request: A `BTPayPalEditRequest`
+    ///   - completion: This completion will be invoked exactly once when tokenization is complete or an error occurs.
+    /// - Warning: This feature is currently in beta and may change or be removed in future releases.
+    @objc(tokenizeWithEditRequest:completion:)
+    public func tokenize(
+        _ request: BTPayPalEditRequest,
+        completion: @escaping (BTPayPalAccountNonce?, Error?) -> Void
+    ) {
+        // TODO: call API to get FI URL and return a BTPayPalNonce or Error
```

Comment:
I am now not sure if this happens in tokenize here after reading the more carefully. I think the edit FI interaction is done on the merchant side maybe and what we do in SDK with our client SDK API is to get token that represents the updated FI and update the nonce?